src/cli/buckets_sample.rs

```diff
@@ -103,32 +86,41 @@ fn load_sample_bucket(
             .http_client()
             .management_request(
                 ManagementRequest::LoadSampleBucket {
-                    name: format!("[\"{}\"]", bucket_name),
+                    name: format!("[\"{}\"]", bucket_name.clone()),
                 },
                 Instant::now().add(cluster.timeouts().management_timeout()),
                 ctrl_c.clone(),
             )
             .map_err(|e| client_error_to_shell_error(e, span))?;
 
+        let mut collected = NuValueMap::default();
+        collected.add_string("cluster", identifier.clone(), span);
+        collected.add_string("sample", bucket_name.clone(), span);
+
         match response.status() {
-            202 => {}
+            202 => {
+                collected.add_string("status", "success", span);
+            }
             _ => {
-                return Err(unexpected_status_code_error(
-                    response.status(),
-                    response.content(),
-                    span,
-                ))
+                // The format of the returned error varies between server versions so we have to do this substring matching
+                let error = if response.content().contains("already loaded") {
```

Comment:
Typically I'd say that string matching is going to lead to pain down the road. Repeatedly we've seen that the server changes the message and error handling gets broken on clients. Admittedly there's not really a better way but can we just continue surfacing the error message from the server directly?